pkg/controller/operator/seed/reconciler_ee_test.go

```diff
@@ -0,0 +1,167 @@
+//go:build ee
```

Comment:
Added wrapper although I have mixed feeling if that should be the way to handle this particular case. Let me know what you think.

Reply:
I mean you could make a test instead in `/ee/metering/reconcile_test.go` and test there without the whole reconcile, just using `ReconcileMeteringResources(...)` , just that it will take a bit more work.

I dont have strong feelings about this, your solution works as well.

Reply:
I see your point, but as you said it would take some additional effort to implement the test. If you don't have a strong feelings about it I would stick to the way it it implemented to safe some time.